content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -305,7 +305,7 @@ into `/var/lib/kubelet/config/init/kubelet` file.
 
 The init configuration is used for starting the kubelet on this specific node, providing an alternative for the kubelet drop-in file;
 such configuration will be replaced by the kubelet base configuration as described in following steps.
-See [set Kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file) for additional info.
+See [set kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file) for additional information.
```

Comment:
some of these changes are valid but adjacent to the introduction of the kubelet component config.

Reply:
Correct. When scanning the page for links to adapt/revise, I saw some trivial defects and took the freedom to fix them. These changes are too trivial to warrant a separate PR.